### PR TITLE
Gough 5

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: history-of-the-quakers-v3/original/20-book-5-chapter-5.adoc [hunk 1]
```diff
@@ -213,16 +220,21 @@ by such a public vindication of your honour, as your own prudence,
...
 I am, with unfeigned and most respectful affection,
 
+[.signed-section-closing]
 Honoured Sir,
 
-Your most humble
-
-And most obedient servant.
+[.signed-section-signature]
+Your most humble and most obedient servant.
```
Comment (maijapapaya) [inline history-of-the-quakers-v3/original/20-book-5-chapter-5.adoc:227]: This seems too long and looks a little funny in the PDF, but idk what else to do.

File: history-of-the-quakers-v3/original/20-book-5-chapter-5.adoc [hunk 2]
```diff
@@ -213,16 +220,21 @@ by such a public vindication of your honour, as your own prudence,
...
 
+[.signed-section-context-close]
 London, October the 20th, 1688.
 
+--
+
+[.offset]
 W+++.+++ Penn`'s Answer to the foregoing Letter.
 
+[.salutation]
```
Comment (jaredh159) [inline history-of-the-quakers-v3/original/20-book-5-chapter-5.adoc:237]: You forgot to open up a new embedded letter here.

File: history-of-the-quakers-v3/original/20-book-5-chapter-5.adoc (hunk 1 above)
Comment (jaredh159) [reply history-of-the-quakers-v3/original/20-book-5-chapter-5.adoc:227]: yeah, this is sorta weird. How about we nix the [ssc] above and move the `Honored Sir,` up onto the same line with the previous phrase (line 221). 

Then, I think the sscc seems pretty good, even though it's a little long. It does seem like it's his "signature" 👍

**Comment by jaredh159** [comment]
Merged. 👍

**Pull request merged**